src/loader/loader.py

```diff
@@ -39,16 +42,17 @@ def fetch_packages(*, pypi_client: PyPIServices) -> list[tuple[str, str]]:
 def load_packages(packages: list[tuple[str, str]], *, http_client: Client, access_token: str) -> None:
     """Load all of the given packages into the Dragonfly API, using the given HTTP session and access token."""
     payload = [{"name": name, "version": version} for name, version in packages]
+    logger.info("Loading packages: %s", payload)
```

Comment:
Considering this will spit out 100 name/value pairs this should probably be debug
```suggestion
    logger.debug("Loading packages: %s", payload)
```